sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/ResponseUtils.java

```diff
@@ -16,11 +19,24 @@ static Mono<StoreResponse> toStoreResponse(HttpResponse httpClientResponse, Http
 
         HttpHeaders httpResponseHeaders = httpClientResponse.headers();
 
-        Mono<byte[]> contentObservable = httpClientResponse.bodyAsByteArray().switchIfEmpty(Mono.just(EMPTY_BYTE_ARRAY));
```

Comment:
not related this PR, but nit: remove used parameter `httpRequest`

Reply:
Fixed